broadinstitute/single_cell_portal_core
# Add titles to unique pages in SCP (SCP-4599)

Page titles are an important piece of helping users quickly orient themselves and scan information but previously all pages on SCP shared the title “Single Cell Portal” this update seeks to add more context to unique pages in SCP.

This adds unique titles to a variety of SCP pages, with a default for any/all to be "Single Cell Portal"

To test:
- pull this branch
- click around and check out the title of the page (visible in the tab and also visible when viewing the back history)
- confirm that the title updates for unique pages like studies and pages available from the user drop down section

<img width="359" alt="Screen Shot 2022-08-17 at 4 13 00 PM" src="https://user-images.githubusercontent.com/54322292/185233977-db6ba378-0b7c-4c99-bb6d-a4b28d4e65b1.png">
